tiny_dnn/util/nms.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (c) 2013, Taiga Nomi and the respective contributors
+  All rights reserved.
+
+  Use of this source code is governed by a BSD-style license that can be found
+  in the LICENSE file.
+*/
+#pragma once
+
+#include <vector>
+
+namespace tiny_dnn {
+
+struct bounding_box {
+  float x_min;
+  float y_min;
+  float x_max;
+  float y_max;
+  float score;
```

Comment:
I guess using `float_t` might be better or you can make this template struct.

https://github.com/tiny-dnn/tiny-dnn/blob/0434d2654b06da73ce318b3ff8f5db9562d8d912/tiny_dnn/config.h#L88-L92